Library/Homebrew/software_spec.rb

```diff
@@ -212,8 +212,9 @@ def patch(strip = :p1, src = nil, &block)
   end
 
   def fails_with(compiler, &block)
-    odisabled "fails_with :llvm" if compiler == :llvm
-    compiler_failures << CompilerFailure.create(compiler, &block)
+    failure = CompilerFailure.create(compiler, &block)
+    odisabled "fails_with #{compiler.inspect}" unless CompilerSelector::COMPILERS.include? failure.name
```

Comment:
Probably want to go for an `odeprecated` first for non-LLVM. Ideally that code would all live in `compat`.